app/server/src/main/scala/org/bitcoins/server/Main.scala

```diff
@@ -38,13 +38,13 @@ object Main extends App {
   val peerSocket =
     parseInetSocketAddress(nodeConf.peers.head, nodeConf.network.port)
   val peer = Peer.fromSocket(peerSocket)
-
+  val bip39PasswordOpt = None //todo need to prompt user for this
```

Comment:
ping, or maybe this needs an issue (or comment on existing #972) if you plan on doing this on a future PR